app/lib/features/settings/pages/labs_page.dart

```diff
@@ -29,11 +29,11 @@ class SettingsLabsPage extends ConsumerWidget {
               ],
             ),
             SettingsSection(
-              title: Text(L10n.of(context).spaces('')),
+              title: Text(L10n.of(context).spaces),
               tiles: [
                 SettingsTile.switchTile(
                   title: Text(
-                    '${L10n.of(context).encrypted} ${L10n.of(context).spaces('').toLowerCase()}',
+                    '${L10n.of(context).encrypted} ${L10n.of(context).spaces.toLowerCase()}',
```

Comment:
grammar: Order of words, forced letter casing ... all that is language specific:

```suggestion
                    L10n.of(context).encryptedSpaces,
```